google/cloud/storage/bucket.py

```diff
@@ -1709,6 +1710,9 @@ def delete_blobs(
 
         Uses :meth:`delete_blob` to delete each individual blob.
 
+        By default, the latest versions of the blobs are deleted. Set `preserve_generation`
```

Comment:
Technically it's the live version, not the latest version. May I suggest the wording, "By default, any generation information in the list of blobs is ignored, and the live versions of all blobs are deleted. Set `preserve_generation` to True if blob generation should instead be propagated from the list of blobs."

Reply:
SGTM👍👍

Agreed "live version" works better here especially in the suggested wording, updating the docstring. Argh I've been seeing both terms floating around in the API docs, and I tried to align wording with the [objects delete doc](https://cloud.google.com/storage/docs/json_api/v1/objects/delete)

Reply:
Yeah, I think we should standardize on live version everywhere if we can. If "latest version", people would expect the behavior to be that you can run this command on a blob twice, once to delete the latest version, and then once more to delete the version before that. But that's not how it works - you run the command on a blob twice and the first time it will delete the live version, and then there is no more live version, so the second time it won't delete anything. You'd need to, indeed, use this new flag to delete the older version deliberately.